### ISSUE TITLE
File extension of handlebar files is not correct.

### ISSUE THREAD

**Issue by frappelatte28** [wecodemore/grunt-githooks#72]
<topic>
File extension of handlebar files is not correct.

### Changes to discuss
File extension `.hb` is misleading GitHub as it is showing harbour programming language instead of handlebars.
It can be `.hbs` or `.handlebars`

**Comment by franz-josef-kaiser** [comment]
@frappelatte28 You are right. But Handlebars should receive an upgrade as well as [seen here](https://github.com/wecodemore/grunt-githooks/security/dependabot/package.json/handlebars/open). I currently have no time to test this. If you want to jump in and lend a helping hand: Every contribution is welcome.
